stan/math/prim/meta/is_container.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef STAN_MATH_PRIM_SCAL_META_IS_CONTAINER_HPP
+#define STAN_MATH_PRIM_SCAL_META_IS_CONTAINER_HPP
```

Comment:
Now that the meta flatten has happened, these header guards should just be be PRIM_META

Reply:
Word!